dxr/plugins/core.py

```diff
@@ -414,9 +414,9 @@ class IdFilter(FilterAggregator):
     name = 'id'
     domain = LINE
     description = Markup('Definition of an identifier: '
-                         '<code>id:someFunction</code> <code>id:SomeClass</code> '
-                         '<code>id:@function</code> '
-                         '(@ forces case-sensitive search, otherwise automatic)')
+                         '<code>id:SomeClass</code> <code>id:@foofunction</code> '
+                         '(@ forces case-sensitive search, '
```

Comment:
Let's stretch it out and be more explicit, since we're already hiding this pretty sneakily: (<code>@</code> forces case-sensitive search. Otherwise, case-sensitive is assumed if there are capital letters in the query.)